ldk/javascript/readme.md

```diff
@@ -28,6 +28,43 @@ const merged = merge.merge(ldkConfig.default, {
 module.exports = merged;
 ```
 
+### Loop Permissions
```

Comment:
One specific thing I think we should consider is that you have to provide a reason for why the loop needs access to some (or all) things. I'm thinking of how Apple does it with apps - if an app wants access to the phone's microphone, the app has to specify why (which will be displayed to the user when the app tries to access the microphone).
So **if** we included this, whether we do it at the aptitude level, or the individual item level (like for each file or URL), the loop reviewers **and** the final users will know why the loop is going to read a specific file or make requests to a certain domain.
Now just like with apps, the dev can totally lie, and there's no proving anything. But it at least gives an opportunity.

Reply:
Yeah, this makes sense to me.

Reply:
Love this idea.